### PR TITLE
filePathRegex

### PR DESCRIPTION
**Pull request by noamshemesh**
Add file path regex parameter to enable template name modifications

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "Include AngularJS templates in the Webpack bundle and preload the template cache.",
   "main": "index.js",
   "scripts": {
-    "test": "./node_modules/webpack/bin/webpack.js --config=test/webpack.config.js"
+    "test": "webpack --config=test/webpack.config.js"
```
Comment (e-cloud) [inline package.json]: it' not reasonable to rely on the global installed webpack

Comment (noamshemesh) [inline package.json]: totally agree, though ./node_modules/.bin is added automatically to PATH when running `npm <command>`.
https://docs.npmjs.com/misc/scripts#path

**Comment by jtomaszewski** [comment]
Useful PR - it helps me f.e. to rename paths to my templates like `templateName.html.haml` or `templateName.html.pug` to `templateName.html` ;)

**Comment by WearyMonkey** [comment]
Thanks for the PR!

The same effect can now be achieved with parameter interpolation in 2.0.0.

**Pull request closed without merging**